pandas/tests/io/test_parquet.py

```diff
@@ -663,6 +664,20 @@ def test_partition_cols_string(self, pa, df_full):
             assert len(dataset.partitions.partition_names) == 1
             assert dataset.partitions.partition_names == set(partition_cols_list)
 
+    def test_partition_cols_pathlib(self, pa, df_compat):
```

Comment:
can you instead parameterize the above test (on path)

Reply:
Done